cmd/releases.go

```diff
@@ -103,8 +109,27 @@ func runReleaseCreate(ctx *cli.Context) error {
 	})
 
 	if err != nil {
+		if err.Error() == "409 Conflict" {
+			log.Fatal("error: There already is a release for this tag")
+		}
+
 		log.Fatal(err)
 	}
 
+	for _, asset := range ctx.StringSlice("asset") {
+		var file *os.File
+
+		if file, err = os.Open(asset); err != nil {
+			log.Fatal(err)
+		}
+		defer file.Close()
```

Comment:
This won't close file until function exits. In loop you must use `file.Close()`  manually where needed

Reply:
Now it shoud be working. Thanks!